3.2/docker-entrypoint.sh

```diff
@@ -28,11 +28,14 @@ case "$1" in
 		if [ ! -f './config/database.yml' ]; then
 			file_env 'REDMINE_DB_MYSQL'
 			file_env 'REDMINE_DB_POSTGRES'
+			file_env 'REDMINE_DB_SQLSERVER'
 			
 			if [ "$MYSQL_PORT_3306_TCP" ] && [ -z "$REDMINE_DB_MYSQL" ]; then
 				export REDMINE_DB_MYSQL='mysql'
 			elif [ "$POSTGRES_PORT_5432_TCP" ] && [ -z "$REDMINE_DB_POSTGRES" ]; then
 				export REDMINE_DB_POSTGRES='postgres'
+			elif [ "$POSTGRES_PORT_1433_TCP" ] && [ -z "$REDMINE_DB_SQLSERVER" ]; then
```

Comment:
`POSTGRES_PORT_1433_TCP` here should be `SQLSERVER_PORT_1433_TCP` instead

Reply:
On second thought, I think we should remove this section -- this section is here for backwards compatibility with `--link`'d containers, and I don't think we need to support that for SQL Server, so this entire `elif` should just go away.

Reply:
Thanks for spotting the copy-pasta there.